tools/gen_openvino_dockerfile.py

```diff
@@ -138,16 +136,15 @@ def dockerfile_for_windows(output_file):
 ARG OPENVINO_BUILD_TYPE
 
 WORKDIR /workspace
-RUN curl -L https://storage.openvinotoolkit.org/repositories/openvino/packages/2023.3/windows/w_openvino_toolkit_windows_2023.3.0.13775.ceeafaf64f3_x86_64.zip --output ov.zip
+RUN curl -L https://storage.openvinotoolkit.org/repositories/openvino/packages/2024.0/windows/w_openvino_toolkit_windows_2024.0.0.14509.34caeefd078_x86_64.zip --output ov.zip
 RUN tar -xf ov.zip
-RUN ren w_openvino_toolkit_windows_2023.3.0.13775.ceeafaf64f3_x86_64 install
+RUN ren w_openvino_toolkit_windows_2024.0.0.14509.34caeefd078_x86_64 install
 
 WORKDIR /opt/openvino
 RUN xcopy /I /E \\workspace\\install\\docs\\licensing LICENSE.openvino
 RUN mkdir include
 RUN xcopy /I /E \\workspace\\install\\runtime\\include\\ie include
```

Comment:
```suggestion
```

Reply:
@ilya-lavrenov - is there a copy command that we could use to copy if the folder exists? ideally we would like to keep the change backwards compatible and select the version via argument ( at least as we move default from 2023.3.0 -> 2024.0.0 ) - reasoning is we can merge this change independently of the build.py changes (i.e. keep build.py pointing to older version and then update and test that once this one is completed).

Reply:
API 1.0 is not used by Triton backend